packages/calcite-components/src/components/date-picker-month/date-picker-month.tsx

```diff
@@ -47,14 +47,14 @@ export class DatePickerMonth {
   /** Already selected date.*/
   @Prop() selectedDate: Date;
 
-  /** Date currently active.*/
+  /** The currently active Date.*/
   @Prop() activeDate: Date = new Date();
 
   /** Start date currently active. */
-  @Prop() startDate?: Date;
+  @Prop() startDate: Date;
 
-  /** End date currently active  */
-  @Prop() endDate?: Date;
+  /** End date currently active.  */
+  @Prop() endDate: Date;
```

Comment:
These optional props _may_ still be needed for `startDate` and `endDate`, can they be added back in?